custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -164,12 +165,30 @@ function DownloadController($scope, $rootScope, $location, locationHierarchy, lo
         vm.indicators.push({id: 12, name: 'Service Delivery Report'});
         vm.indicators.push({id: 13, name: 'Child Growth Tracking Report'});
         vm.indicators.push({id: 14, name: 'AWW Activity Report'});
-
+        vm.indicators.push({id: 15, name: 'Poshan Progress Report'});
+        vm.reportLayouts = [
+            {id: 'comprehensive', name: 'Comprehensive'},
+            {id: 'summary', name: 'Summary'},
+        ];
+        vm.dataPeriods = [
+            {id: 1, name: 'Monthly'},
+            {id: 3, name: 'Quarterly'},
```

Comment:
any specific reason to use 1 and 3? not 1 and 2

Reply:
1 -> one month at a time and 3-> 3 months at a time if half yearly comes 6 -> 6 months at a time.